dist_tables/dml.rst

```diff
@@ -38,10 +38,98 @@ For example:
 
     If COPY fails to open a connection for a shard placement then it behaves in the same way as INSERT, namely to mark the placement(s) as inactive unless there are no more active placements. If any other failure occurs after connecting, the transaction is rolled back and thus no metadata changes are made.
 
+Distributed Aggregations
+$$$$$$$$$$$$$$$$$$$$$$$$
+
+Applications like event data pipelines and real-time dashboards require fast queries across predefined aggregations of incoming data. One way to make these queries fast is by calculating and saving aggregates ahead of time. This is called "rolling up" the data and it avoids the cost of processing raw data at run-time. As an extra benefit, rolling up timeseries data into hourly or daily statistics can also save space. Old data may be deleted when its full details are no longer needed and aggregates suffice.
```

Comment:
Minor edit.

... require fast queries across predefined aggregations of incoming data. -> ... require sub-second queries on large volumes of data.